theano/scan_module/scan.py

```diff
@@ -314,6 +316,10 @@ def f(x):
         Set the value of allow gc for the internal graph of scan.  If
         set to None, this will use the value of config.scan.allow_gc.
 
+    :param strict:
+        If true, all the shared variables used in ``fn`` must be provided as a
+        part of ``non_sequences``. 
```

Comment:
On Mon, Mar 16, 2015 at 4:04 PM, Pascal Lamblin <notifications@github.com>
wrote:

> In theano/scan_module/scan.py
> <https://github.com/Theano/Theano/pull/2622#discussion_r26522029>:
>
> > @@ -314,6 +316,10 @@ def f(x):
> >          Set the value of allow gc for the internal graph of scan.  If
> >          set to None, this will use the value of config.scan.allow_gc.
> >
> > +    :param strict:
> > +        If true, all the shared variables used in ``fn`` must be provided as a
> > +        part of ``non_sequences``.
>
> or sequences.
>
Fixed!


> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2622/files#r26522029>.
>